handout/testing/services/injecting-dependencies.md

```diff
@@ -1,3 +1,3 @@
 # Injecting Dependencies
 
-Like components, services often require dependencies that Angular injects through the constructor of the service's class. Since we are initializing these classes outside the bootstrapping process of Angular, we must explicitly inject these dependencies ourselves. This is accomplished by calling `addProviders` in the `beforeEach` part of your unit test to feed in required dependencies like the HTTP module.
+Like components, services often require dependencies that Angular injects through the constructor of the service's class. Since we are initializing these classes outside the bootstrapping process of Angular, we must explicitly inject these dependencies ourselves. This is accomplished by using the `TestBed` to configure a testing module and feed in required dependencies like the HTTP module.
```

Comment:
I was reading at the book and I noticed that there are two consecutive chapters that only have one paragraph: "Testing Strategies for Services" and "Injecting Dependencies". How do you feel about merging them in just one chapter?